strongbox-commons/src/main/java/org/carlspring/strongbox/io/ReplacingInputStream.java

```diff
@@ -0,0 +1,102 @@
+package org.carlspring.strongbox.io;
+
+import java.io.FilterInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Deque;
+import java.util.Iterator;
+import java.util.LinkedList;
+
+
+public class ReplacingInputStream extends FilterInputStream {
```

Comment:
it used to replace missing bytes in Nuget multipart stream, I thought that the meaning is clear with class name :)
but i will provide javadocs if you point for this of course